apps/examples/wifi_manager_sample/wm_test_stress_test3.c

```diff
@@ -0,0 +1,288 @@
+	wm_get_softapinfo(&softap_config);
+	wres = wifi_manager_set_mode(SOFTAP_MODE, &softap_config);
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "fail to start softap %d\n", wres);
+		return -1;
+	}
+
+	/*  wait join event */
+	WT_LOG(TAG, "wait join event");
+
+	WM_TEST_WAIT;
+
+	/*  scan in softAP mode */
+	WT_LOG(TAG, "scan in softAP mode");
+
+	wres = wifi_manager_scan_ap();
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "fail to scan %d\n", wres);
+		return -1;
+	}
+
+	/*  wait scan event */
+	WT_LOG(TAG, "wait scan done event");
+
+	WM_TEST_WAIT;
+
+	/*  send data */
+	WT_LOG(TAG, "send dummy data size %d", WT_DATA_SIZE);
+
+	nres = wt_send_dummy(WT_DATA_SIZE);
+	if (nres < 0) {
+		WT_LOGE(TAG, "send dummy data fail %d\n", nres);
+		return -1;
+	}
+
+	/*  set STA */
+	WT_LOG(TAG, "start STA mode");
+
+	wres = wifi_manager_set_mode(STA_MODE, NULL);
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "start STA fail %d\n", wres);
+		return -1;
+	}
+
+	/*  scan in STA mode */
+	WT_LOG(TAG, "scan in STA mode");
+
+	wres = wifi_manager_scan_ap();
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "fail to scan %d\n", wres);
+		return -1;
+	}
+	WT_LOG(TAG, "wait scan done event in STA mode");
+
+	WM_TEST_WAIT; /*  wait scan event */
+
+	/*  connect to AP */
+	WT_LOG(TAG, "connect AP");
+
+	wifi_manager_ap_config_s apconfig;
+	wm_get_apinfo(&apconfig);
+	wres = wifi_manager_connect_ap(&apconfig);
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "connect AP fail %d\n", wres);
+		return -1;
+	}
+	WT_LOG(TAG, "wait connect success event");
+
+	WM_TEST_WAIT;
+
+	/*  scan in connected state */
+	WT_LOG(TAG, "scan in connected state of STA mode");
+
+	wres = wifi_manager_scan_ap();
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "fail to scan %d\n", wres);
+		return -1;
+	}
+	WT_LOG(TAG, "wait scan done event in connected state of STA mode");
+
+	WM_TEST_WAIT; /*  wait scan event */
+
+	/*  send data */
+	WT_LOG(TAG, "send dummy data %d", WT_DATA_SIZE);
+
+	nres = wt_send_dummy(WT_DATA_SIZE);
+	if (nres < 0) {
+		WT_LOGE(TAG, "send dummy data fail %d\n", nres);
+		return -1;
+	}
+
+	WT_LOG(TAG, "deinit wi-fi");
+
+	wres = wifi_manager_deinit();
+	if (wres != WIFI_MANAGER_SUCCESS) {
+		WT_LOGE(TAG, "fail to deinit %d\n", wres);
+		return -1;
+	}
+	return 0;
+}
+
+TEST_F(mode_change)
+{
+	ST_START_TEST;
+	ST_EXPECT_EQ(0, _run_procedure());
+	ST_END_TEST;
+}
+
+void wm_run_stress_test3(struct wt_options *opt)
+{
+	WM_AP_SSID = opt->ssid;
+	WM_AP_PASSWORD = opt->password;
+	WM_AP_AUTH = opt->auth_type;
+	WM_AP_CRYPTO = opt->crypto_type;
+	WM_SOFTAP_SSID = opt->softap_ssid;
+	WM_SOFTAP_PASSWORD = opt->softap_password;
+	WM_SOFTAP_CHANNEL = opt->softap_channel;
+
+	WT_LOG(TAG, "init sem");
+	if (0 != sem_init(&g_wm_sem, 0, 0)) {
+		WT_LOGE(TAG, "sem init fail %d", errno);
+		return;
+	}
+	ST_SET_PACK(wifi);
+
+	ST_SET_SMOKE1(wifi, WM_TEST_TRIAL, 10000000, "use case test", mode_change);
```

Comment:
@pillip8282 
In default defconfigs, 'perf_add_item' is not built because it is not enabled.
So you need to enable 'stress_tool' in all defconfigs to use it